### DISCUSSION TITLE
Widget Repository?

### DISCUSSION THREAD

**Discussion by IllustratedMan-code** [elkowar/eww#536] [Ideas]
I think it would be great to have a place to put widgets (and or windows) for the community. I think this would make it a lot easier for people to get started with eww.

Thoughts?

**Comment by VuiMuich** [comment]
I like this idea. If I may share some experience from [leftwm/leftwm-community-themes](https://github.com/leftwm/leftwm-community-themes), don't go with submodules it sucks balls, when you would need to patch small changes that almost all widgets are affected of. E.g. if `defwindow` would be renamed to `defwin`.
